Specification/8._Hardware_design_recommendations.md

```diff
@@ -288,17 +286,17 @@ not including additional delay times of CAN transceivers and other components.
 
 #### Automatic bit rate detection
 
-It is encouraged to implement CAN auto bit rate detection when applicable.
+Designers are encouraged to implement CAN auto bit rate detection when applicable.
 Please refer to the CiA 801 application note for recommended practices.
 
-It should be noted that UAVCAN enables the use of the simple bit time measuring approach,
+UAVCAN enables the use of the simple bit time measuring approach,
 as it is guaranteed that any functioning UAVCAN network will always exchange node status messages,
 which can be expected to be published at a rate no lower than 1 Hz,
 and that contain a suitable alternating bit pattern in the CAN ID field.
 Please refer to the [chapter dedicated to the standard data types](/Specification/6._Application_level_functions)
-for details.
+for details. 
 
 ## See also
 
-* <http://www.ti.com/lit/an/slla270/slla270.pdf>
-* <http://www.computer-solutions.co.uk/info/Embedded_tutorials/can_tutorial.htm>
+* <http://www.ti.com/lit/an/slla270/slla270.pdf> 
```

Comment:
I quite like links that have text/descriptions so that I know what it is that I might be choosing to read. Add description?

Reply:
If you think that that will help, sure.